src/test/java/TestPrinterFormatter.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.Test;
+import training.supportbank.PrinterFormatter;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class TestPrinterFormatter {
+    private static Map<String, BigDecimal> fakeMapInput = new LinkedHashMap<>();
+    private static BigDecimal fakeBigDecimal;
+
+    @Test
+    public void formatterConvertToCurrencyOutputsStringCurrencyValue() {
+        fakeBigDecimal = new BigDecimal("5.00");
```

Comment:
Very minor point, but I'd call this variable something like `money` or `fivePounds`. "Fake" makes it sound like it's a test double that isn't a real `BigDecimal`, but in this case it is a real value. 🙂

Reply:
Resolved in next comment.